srcpkgs/cmake-bootstrap/patches/only-want-half-baked-cmake.patch

```diff
@@ -1,122 +0,0 @@
-diff --git Source/cmSystemTools.cxx Source/cmSystemTools.cxx
-index 1e78d36..a678122 100644
---- Source/cmSystemTools.cxx
-+++ Source/cmSystemTools.cxx
-@@ -1988,7 +1988,7 @@ void cmSystemTools::FindCMakeResources(const char* argv0)
- #ifdef CMAKE_BOOTSTRAP
-   // The bootstrap cmake does not provide the other tools,
-   // so use the directory where they are about to be built.
--  exe_dir = CMAKE_BOOTSTRAP_BINARY_DIR "/bin";
-+  // this cmake will be installed to system directory
- #endif
-   cmSystemToolsCTestCommand =
-     cmStrCat(exe_dir, "/ctest", cmSystemTools::GetExecutableExtension());
-@@ -2010,7 +2010,6 @@ void cmSystemTools::FindCMakeResources(const char* argv0)
-     cmSystemToolsCMClDepsCommand.clear();
-   }
- 
--#ifndef CMAKE_BOOTSTRAP
-   // Install tree has
-   // - "<prefix><CMAKE_BIN_DIR>/cmake"
```

Comment:
Did you run the full bootstrap with this change?

Reply:
If it non-cross-build it builds fine; Currently I am trying to fix cross, then test cmake and cmake-gui against it.

Reply:
Why does it have problem with cross build? The binaries should never run at all.
And why do you remove this patch (to C++ code).

Reply:
Let me try and remake the patch since it fails to apply on this version